modules/microservice/main.tf

```diff
@@ -4,18 +4,18 @@
 module "service" {
   source = "../service"
 
-  name_prefix                               = "${var.name_prefix}"
-  vpc_id                                    = "${var.vpc_id}"
-  cluster_id                                = "${var.cluster_id}"
-  cluster_role_name                         = "${var.cluster_role_name}"
-  desired_count                             = "${var.desired_count}"
-  task_definition_cpu                       = "${var.task_definition_cpu}"
-  task_definition_memory_reservation        = "${var.task_definition_memory_reservation}"
-  task_definition_image                     = "${var.task_definition_image}"
-  task_definition_command                   = "${var.task_definition_command}"
-  task_definition_environment               = "${var.task_definition_environment}"
-  task_definition_health_check_grace_period = "${var.task_definition_health_check_grace_period}"
-  health                                    = "${var.health}"
+  name_prefix                       = "${var.name_prefix}"
+  vpc_id                            = "${var.vpc_id}"
+  cluster_id                        = "${var.cluster_id}"
+  cluster_role_name                 = "${var.cluster_role_name}"
+  desired_count                     = "${var.desired_count}"
+  task_container_cpu                = "${var.task_container_cpu}"
+  task_container_memory_reservation = "${var.task_container_memory_reservation}"
+  task_container_image              = "${var.task_container_image}"
+  task_container_command            = "${var.task_container_command}"
+  task_container_environment        = "${var.task_container_environment}"
+  health                            = "${var.health}"
```

Comment:
To be consistent this should be renamed to `health_check` instead of `health`

Reply:
Good catch! Let me know if you see anything else - hard to spot the renames in the PR since it's the first commit.